vimrc

```diff
@@ -145,13 +145,15 @@ endfunction
 
 vnoremap * :<C-u>call <SID>VSetSearch()<CR>/<CR>
 
-let g:ale_enabled = 0                     " Disable linting by default
+let g:ale_enabled = 1                     " Disable linting by default
```

Comment:
"Enable linting by default"

Speaking of which, if this is enabled by default, I assume that it would lint the entire file? Would that be a concern, or overwhelming, for large files (e.g. Transaction model in the Gateway)?

Reply:
Unfortunately ale doesn't support ranges of files, so it would work on an entire file and not just on or around changed lines.

Reply:
@eanlain I tried loading our largest ruby file and I didn't experience anything out of the ordinary. It contained some violations, but I could scroll through it and interact with it just fine.